Build - PHPLoc repository no longer responds

issue reported by @jeremychauvet
> Hello !
> http://repo.phloc.com/maven2/ is not a valid URL anymore. 
> Could you update it ?
> Thanks !

## Observed behaviour

- PHPLoc repository no longer responds 
- The central repository only contains more recent versions 
of [phloc-commons](https://mvnrepository.com/artifact/com.phloc/phloc-commons) and [phloc-css](https://mvnrepository.com/artifact/com.phloc/phloc-css) than those used by _Asqatsun_.

see [pom.xml file - Line 141](https://github.com/Asqatasun/Asqatasun/blob/develop/pom.xml#L141)

```
        <repository>
            <id>phlocRepos</id>
            <url>http://repo.phloc.com/maven2/</url>
```

see [pom.xml file - Line 1 548](https://github.com/Asqatasun/Asqatasun/blob/develop/pom.xml#L1548)
```
            <dependency>
                <groupId>com.phloc</groupId>
                <artifactId>phloc-css</artifactId>
                <version>3.6.0</version>
            </dependency>
            <dependency>
                <groupId>com.phloc</groupId>
                <artifactId>phloc-commons</artifactId>
                <version>4.0.11</version>
            </dependency>
```

## Steps to reproduce the behaviour
```
mv ~/.m2 ~/.m2_backup
mvn clean install

(...)
[INFO] ------------------------------------------------------------------------
[INFO] Building asqatasun-api 4.1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[WARNING] The POM for com.phloc:phloc-css:jar:3.6.0 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] asqatasun .......................................... SUCCESS [  0.895 s]
[INFO] engine ............................................. SUCCESS [  0.010 s]
[INFO] commons-entity ..................................... SUCCESS [  0.719 s]
[INFO] commons-entity-jpa ................................. SUCCESS [  0.089 s]
[INFO] asqatasun-api ...................................... FAILURE [  0.043 s]

[ERROR] Failed to execute goal on project asqatasun-api: Could not resolve dependencies for project org.asqatasun:asqatasun-api:jar:4.1.0-SNAPSHOT: 
Failure to find com.phloc:phloc-css:jar:3.6.0 in http://repo1.maven.org/maven2/ 
was cached in the local repository, resolution will not be reattempted until 
the update interval of central has elapsed or updates are forced 
```

--- a/pom.xml
+++ b/pom.xml
@@ -137,16 +137,6 @@
             </snapshots>
         </repository>
         <repository>
-            <id>phlocRepos</id>
-            <url>http://repo.phloc.com/maven2/</url>
-            <releases>
-                <enabled>true</enabled>
-            </releases>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-        </repository>
-        <repository>
             <id>heritrixRepos</id>
             <url>http://builds.archive.org:8080/maven2/</url>
             <releases>
@@ -1545,12 +1535,12 @@
             <dependency>
                 <groupId>com.phloc</groupId>
                 <artifactId>phloc-css</artifactId>
-                <version>3.6.0</version>
+                <version>3.8.0</version>
             </dependency>
             <dependency>
                 <groupId>com.phloc</groupId>
                 <artifactId>phloc-commons</artifactId>
-                <version>4.0.11</version>
+                <version>4.4.13</version>
             </dependency>
             <dependency>
                 <groupId>org.json</groupId>

